NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -308,7 +308,7 @@ public class BGoldNodeDownloadData
 				Mac = new NodeOSDownloadData()
 				{
 					DownloadLink = "https://github.com/BTCGPU/BTCGPU/releases/download/v{0}.2/bitcoin-gold-{0}-osx64.tar.gz",
-					Archive = "bitcoin-gold-{0}-osx64.tar.gz",
+				        Archive = "bitcoin-gold-{0}-osx64.tar.gz",
```

Comment:
whitespace change not related to BitCore

Reply:
only cosmetic changes 
Should I take out them?